src/test/java/StringTest.java

```diff
@@ -0,0 +1,66 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @Test
+    void requirements_1() {
+        //given
+        String input1 = "1,2";
+        String input2 = "1";
+
+        String[] expected1 = {"1", "2"};
+        String[] expected2 = {"1"};
+
+        //when
+        String[] actual1 = input1.split(",");
```

Comment:
```suggestion
        String[] actual1 = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?